build.sbt

```diff
@@ -269,17 +266,49 @@ lazy val `anorm-akka` = (project in file("akka"))
         mimaPreviousArtifacts.value
       }
     },
-    libraryDependencies ++= Seq("akka-testkit", "akka-stream").map { m =>
+    libraryDependencies ++= Seq("akka-testkit", "akka-stream", "akka-stream-testkit").map { m =>
```

Comment:
This must not be there, it's not a `Provided` dependency, but a `Test` one

Reply:
Fixed. 👍